Macros/ACS_summary_geo.sas

```diff
@@ -573,63 +555,242 @@
 	   mhshldinc150000to199999_&_years.
 	   mhshldinc200000andover_&_years.
 
-           ;
+	   mAggFamilyIncome_&_years. 
+
+	   mFamIncomeLT75k_&_years. mFamIncomeGT200k_&_years.
+

```

Comment:
i would prefer to see these vars put into groupings by topic.  - they'll be impossible to find later.